app/generators/Generator.scala

```diff
@@ -38,4 +38,9 @@ class Generator(clock: Clock) {
     now.toLocalTime
       .format(Format.timeFormatter)
 
+  val isoDateTimeLength = 19 // don't include the decimal part of seconds
+  def isoDateTime       = now.format(Format.isoDateTimeFormatter).substring(0, isoDateTimeLength)
```

Comment:
The isoDateTimeFormatter includes nanoseconds, if we modify it to DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss") then we don't have to substring.

Reply:
Yep - I can change to that.